packages/commons/src/authentication.ts

```diff
@@ -62,17 +62,14 @@ const systemScopes = {
   recordsearch: 'recordsearch',
   declare: 'declare',
   notificationApi: 'notification-api',
-  validatorApi: 'validator-api',
-  ageVerificationApi: 'age-verification-api',
   webhook: 'webhook',
   nationalId: 'nationalId'
 } as const
 
 export const systemRoleScopes = {
   HEALTH: [systemScopes.declare, systemScopes.notificationApi],
   NATIONAL_ID: [systemScopes.nationalId],
-  EXTERNAL_VALIDATION: [systemScopes.validatorApi],
-  AGE_CHECK: [systemScopes.declare, systemScopes.ageVerificationApi],
+  AGE_CHECK: [systemScopes.declare],
```

Comment:
I think we can remove this `AGE_CHECK` systemRoleScope completely?

Reply:
Sure